nav2_costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -454,18 +453,15 @@ InflationLayer::dynamicParametersCallback(
         need_reinflation_ = true;
         current_ = false;
       } else if (param_name == name_ + "." + "inflate_unknown" &&
-        inflate_unknown_ != parameter.as_bool())
-      {
+        inflate_unknown_ != parameter.as_bool()) {
```

Comment:
There are CI linting errors introduced from these changes you can see in Circle

Reply:
So, for now I will leave the `ament_cpplint` satisfied or the `ament_uncrustify`?

Reply:
Please make CI happy, that's more important so that we have a good understanding of issues when PRs are submitted

Reply:
Alright! I will do that 😃